[2025-07-08T13:51:13.109+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-08T13:51:13.243+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bash_call_python_file.load_data manual__2025-07-08T13:45:46.798978+00:00 [queued]>
[2025-07-08T13:51:13.278+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bash_call_python_file.load_data manual__2025-07-08T13:45:46.798978+00:00 [queued]>
[2025-07-08T13:51:13.279+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2025-07-08T13:51:13.596+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_data> on 2025-07-08 13:45:46.798978+00:00
[2025-07-08T13:51:13.617+0000] {standard_task_runner.py:63} INFO - Started process 27 to run task
[2025-07-08T13:51:13.648+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'bash_call_python_file', 'load_data', 'manual__2025-07-08T13:45:46.798978+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmpfblfo8wl']
[2025-07-08T13:51:13.663+0000] {standard_task_runner.py:91} INFO - Job 5: Subtask load_data
[2025-07-08T13:51:14.944+0000] {task_command.py:426} INFO - Running <TaskInstance: bash_call_python_file.load_data manual__2025-07-08T13:45:46.798978+00:00 [running]> on host d4707c258089
[2025-07-08T13:51:16.621+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='bash_call_python_file' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-08T13:45:46.798978+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-08T13:45:46.798978+00:00'
[2025-07-08T13:51:16.637+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-08T13:51:16.788+0000] {python.py:237} INFO - Done. Returned value was: This is a sample statement.
[2025-07-08T13:51:16.789+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-08T13:51:16.994+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=bash_call_python_file, task_id=load_data, run_id=manual__2025-07-08T13:45:46.798978+00:00, execution_date=20250708T134546, start_date=20250708T135113, end_date=20250708T135116
[2025-07-08T13:51:17.233+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-08T13:51:17.291+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-08T13:51:17.294+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
